apps/music/js/ui/views/tiles_view.js

```diff
@@ -33,15 +33,17 @@ var TilesView = {
     this.handle = null;
     this.dataSource = [];
     this.index = 0;
+    this.hideSearchTimeout = null;
 
     this.view.addEventListener('click', this);
     this.view.addEventListener('input', this);
     this.view.addEventListener('touchend', this);
+    this.view.addEventListener('scroll', this.cancelHideSearch.bind(this));
     this.searchInput.addEventListener('focus', this);
     this.searchInput.addEventListener('keypress', this);
   },
 
-  clean: function tv_clean() {
+  clean: function tv_clean(ignoreScroll) {
```

Comment:
TilesView.clean() is only called in music.js and do you think it's necessary to have `ignoreScroll` for options? because it should be always true in this patch.